### DISCUSSION TITLE
How to make resizable use % instead of px?

### DISCUSSION THREAD

**Discussion by JuniorVezintana** [GrapesJS/grapesjs#5222] [Q&A]
The default behavior is that when resizing a block as a div, the height and width are in px, I would like to do it in %.

**Comment by artf** [comment]
You can pass [Resizer options](https://github.com/GrapesJS/grapesjs/blob/8ddb9377cf329a9b314be7cab9437bfed730ae2c/src/utils/Resizer.ts#L91-L97) to [resizable component property](https://grapesjs.com/docs/api/component.html#properties)
